templates/webapi/test/settings.html.ep

```diff
@@ -4,9 +4,21 @@
     </thead>
     <tbody>
         % my $s = $job->settings_hash;
+        % my $links = app->config->{template_settings_ui_links};
         % for my $k (sort keys %$s) {
-            % my $v = $s->{$k};
-            <tr><td><%= $k %></td><td><%= $v =~ m{^https?://} ? link_to $v => $v : $v %></td></tr>
+          % my $v = $s->{$k};
+          <tr><td><%= $k %></td>
+          % if ($v =~ m{^https?://}) {
+              <td><%=  link_to $v => $v %></td></tr>
+          % }
+          % else {
+            % if ($links && $v =~ m{^$links}) {
+                <td><%=  link_to $v => $job->id . "/" . $v %></td></tr>
+            % }
+              % else {
```

Comment:
The indentation is off. I would also avoid repeating the last `</tr>` in all the cases.

I would personally prefer if this was kept as an one-liner within the template and instead the `$v =~ m{^https?://} ? link_to $v => $v : $v` part would be extracted as a helper. Note that at one point we intentionally removed all the indentation/white-spaces for better behavior when copying settings to the clipboard.

Reply:
i fixed the indentation. hopefully it is correct now. i also sorted out the `</tr>` repetition. 
The helper sounds right here, but i did not implemented. let me know if you really want it.

Reply:
I actually want the helper. It would leave the template as simple as it was before and would move all the `if`-`else` logic into a dedicated place with a (hopefully) good name.